bin/run-fixed-population

```diff
@@ -196,11 +182,11 @@ if __name__ == '__main__':
         np.random.seed(rand_seed)
 
         # Select the initial binary sample method from user input
-        if args.initial_samp == 'independent':
-            IBT, mass_singles, mass_binaries, n_singles, n_binaries = InitialBinaryTable.sampler(args.initial_samp, kstar1_range, kstar2_range, 0.5, 'kroupa93', 'thermal', SFH_model, component_age, args.metallicity, args.Nstep)
+        if sampling['initial_samp'] == 'independent':
```

Comment:
Should this really be called 'initial_samp'? I know we're sampling the initial conditions, but I think this might be more transparent if it's called 'samp_method' or something.